### PR TITLE
Make Clone URLs region specific

### PR DESCRIPTION
**Pull request by vignesh-goutham**
*Description of changes:*
Make Clone URLs region specific.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/135#" title="Author self-approved">vignesh-goutham</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [vignesh-goutham]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: projects/kubernetes/cloud-provider-vsphere/Makefile
Comment (abhay-krishna) [inline projects/kubernetes/cloud-provider-vsphere/Makefile:7]:
```suggestion
AWS_REGION?=us-west-2
```

**Review by jaxesn — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/135#pullrequestreview-794777662" title="LGTM">jaxesn</a>*, *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/135#" title="Author self-approved">vignesh-goutham</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [jaxesn,vignesh-goutham]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by abhay-krishna — APPROVED** [review]
/lgtm

**Pull request merged**
